src/components/screens/management-screen.tsx

```diff
@@ -84,6 +91,46 @@ const ManagementScreen: React.FC = () => {
     setIsLoading(false);
   };
 
+  /**
+   * Event handler for billing rate update button click
+   */
+  const handleBillingRateUpdateClick = async () => {
+    if (!selectedPersonWithTotalTime || !selectedPersonWithTotalTime.personTotalTime) {
+      return null;
+    }
+
+    if (newBillablePercentage >= 0 && newBillablePercentage <= 100) {
+      const { person } = selectedPersonWithTotalTime;
+      setBillableHoursUpdate(false);
+
+      try {
+        const updatedPerson: PersonWithTotalTime = {
+          person: await Api.getPersonsApi(accessToken?.access_token).updatePerson({
+            personId: person.id,
+            person: { ...person, minimumBillableRate: newBillablePercentage }
+          }),
+          personTotalTime: selectedPersonWithTotalTime.personTotalTime
+        };
+        syncOrUpdateContext.setSyncOrUpdate(strings.billableHoursHandling.updateBillableHoursSuccess);
+        const personIndex = personsTotalTime.findIndex(_person => _person.person.id === person.id);
```

Comment:
Don't use _ prefix. This usually indicates that the variable is unused.

Reply:
Done